Description: Do not set rpath
Author: Ansgar Burchardt <ansgar@debian.org>
Origin: vendor

Do not set rpath for Boost libraries.  While at it, also disable silent rules
to make finding problems easier.

--- cclive-0.7.8.orig/configure.ac
+++ cclive-0.7.8/configure.ac
@@ -21,7 +21,7 @@ AC_DEFINE_UNQUOTED([CANONICAL_TARGET], "
 AC_DEFINE_UNQUOTED([CANONICAL_HOST], "$host", [...])
 
 AM_INIT_AUTOMAKE([-Wall -Werror foreign dist-bzip2 dist-xz])
-AM_SILENT_RULES([yes])
+AM_SILENT_RULES([no])
 
 LT_INIT()
 LT_PREREQ([2.2])
--- cclive-0.7.8.orig/m4/boost.m4
+++ cclive-0.7.8/m4/boost.m4
@@ -403,7 +403,7 @@ dnl generated only once above (before we
       LDFLAGS=$boost_save_LDFLAGS
       LIBS=$boost_save_LIBS
       if test x"$Boost_lib" = xyes; then
-        Boost_lib_LDFLAGS="-L$boost_ldpath -Wl,-R$boost_ldpath"
+        Boost_lib_LDFLAGS="-L$boost_ldpath"
         Boost_lib_LDPATH="$boost_ldpath"
         break 6
       else
